plugins/dynamic-plugins-info/src/components/InternalPluginsMap.tsx

```diff
@@ -0,0 +1,126 @@
+import { DynamicPluginInfo } from '../api/types';
+
+// This is a mapping of internal plugins to their package path based off dynamic-plugins.default.yaml
+export const InternalPluginsMap: Record<string, string> = {
+  'backstage-plugin-scaffolder-backend-module-github-dynamic':
+    './dynamic-plugins/dist/backstage-plugin-scaffolder-backend-module-github-dynamic',
+  'backstage-plugin-catalog-backend-module-github-dynamic':
+    './dynamic-plugins/dist/backstage-plugin-catalog-backend-module-github-dynamic',
+  'backstage-plugin-catalog-backend-module-github-org-dynamic':
+    './dynamic-plugins/dist/backstage-plugin-catalog-backend-module-github-org-dynamic',
+  'backstage-plugin-github-actions':
+    './dynamic-plugins/dist/backstage-plugin-github-actions',
+  'backstage-plugin-github-issues':
+    './dynamic-plugins/dist/backstage-plugin-github-issues',
+  'roadiehq-backstage-plugin-github-insights':
+    './dynamic-plugins/dist/roadiehq-backstage-plugin-github-insights',
+  'roadiehq-backstage-plugin-github-pull-requests':
+    './dynamic-plugins/dist/roadiehq-backstage-plugin-github-pull-requests',
+  'roadiehq-backstage-plugin-security-insights':
+    'roadiehq-backstage-plugin-security-insights',
```

Comment:
Is it expected that this one is not in `./dynamic-plugins/dist` ?

Reply:
@davidfestal Thanks for the review. Good catch on this typo. Will fix it.